src/rocm_docs/rocm_docs_theme/sections/footer.html

```diff
@@ -10,8 +10,7 @@
                 <div class="col-12 text-center">
                     <ul>
                         <li><a href="https://www.amd.com/en/corporate/copyright" target="_blank">Terms and Conditions</a></li>
-                        {% set text, target = license_link %}
-                        <li><a href="{{target}}">{{text}}</a></li>
+                        {{ license_link() }}
```

Comment:
What about copyright? Not required? Did we check with Legal?

Reply:
this is not a project associate with ROCm software and does not need ROCm license information.